edit/sections-editor-section.js

```diff
@@ -60,13 +60,32 @@ function createResizeGrip(cm) {
       const sectionEl = wrapper.parentNode;
       const sectionExtrasHeight = sectionEl.clientHeight - wrapper.offsetHeight;
       cm.state.toggleHeightSaved = wrapper.clientHeight;
-      cm.setSize(null, window.innerHeight - sectionExtrasHeight - pageExtrasHeight);
+      cm.setSize(null, window.innerHeight - sectionExtrasHeight - pageExtrasHeight - 2);
       const bounds = sectionEl.getBoundingClientRect();
       if (bounds.top < 0 || bounds.bottom > window.innerHeight) {
         window.scrollBy(0, bounds.top);
       }
     }
   }
+
+  document.onreadystatechange = () => {
+    if (document.readyState !== 'loading') {
+      new MutationObserver((_, observer) => {
```

Comment:
Why using mutation observer?

Reply:
Wasn't my first choice, but it was the fastest and most reliable of everything I tried. `DOMContentLoaded` was unreliable from one edit page load to another, sometimes it worked, sometimes it didn't. The only `readyState` that worked was 'complete', and it was noticeably late. 

JS isn't my specialty, so if there's anything else as fast and reliable, feel free to hook it up. It needs to get to the calculation as quick as possible, so the delay isn't any longer than it needs to be. With the MO, it's hardly noticeable.